src/mame/konami/gticlub.cpp

```diff
@@ -420,7 +413,7 @@ uint8_t gticlub_base_state::sysreg_r(offs_t offset)
 		}
 
 		default:
-			osd_printf_debug("sysreg_r %d\n", offset);
+			logerror("sysreg_r %d\n", offset);
```

Comment:
Same as the other comment about `osd_printf_debug` – is this hit frequently enough to have performance implications that would make it worth making it conditional?